generator/polygonizer.hpp

```diff
@@ -119,17 +122,15 @@ namespace feature
       {
       case 0:
         break;
-      case 1:
-        EmitFeature(vec[0], fb);
-        break;
+      case 1: EmitFeature(vec[0], fb, m_featureIdToOsmIds); break;
       default:
         {
 #if PARALLEL_POLYGONIZER
           m_ThreadPoolSemaphore.acquire();
-          m_ThreadPool.start(new PolygonizerTask(this, vec, fb));
+          m_ThreadPool.start(new PolygonizerTask(this, vec, fb, m_featureIdToOsmIds));
```

Comment:
Before or after? :)

m_ThreadPool depends on m_featureIdToOsmIds, it's better declared it after m_featureIdToOsmIds.